lib/linux/os_release.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package linux
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/gravitational/trace"
+)
+
+// OSRelease represents the information contained in the /etc/os-release file.
+type OSRelease struct {
```

Comment:
I'm pretty sure there are other parts of teleport that already parse os-release. Might be good to leave a TODO somewhere to consolidate.

Reply:
I found one Go code occurrence (plenty more in other places). Added a TODO for now, I might follow through tomorrow.